aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -13,22 +16,50 @@ use move_binary_format::errors::{PartialVMError, PartialVMResult};
 use move_core_types::account_address::AccountAddress;
 use std::collections::{BTreeMap, BTreeSet};
 
+/// Describes how the `speculative_start_value` in
+/// `AggregatorState` is obtained.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum SpeculativeStartValue {
+    // The speculative_start_value is assigned to 0.
+    Unset,
+    // The speculative_start_value is obtained by reading
+    // the last committed value of the aggregator from MVHashmap.
+    LastCommittedValue(u128),
+    // The speculative_start_value is obtained by aggregating all
```

Comment:
No need to say anything about MVHashMap here (separate abstraction), we can instead say "by performing a read procedure on the aggregator, which involves aggregating deltas present at the read time"